packages/flutter/test/widgets/selectable_region_context_menu_test.dart

```diff
@@ -14,15 +14,10 @@ import 'package:flutter_test/flutter_test.dart';
 import 'package:web/web.dart' as web;
 
 extension on web.HTMLCollection {
-  Iterable<web.Element> get iterable => _genIterable(this);
+  Iterable<web.Element?> get iterable => Iterable<web.Element?>.generate(length, (int index) => item(index));
```

Comment:
Because collections like these can theoretically have holes in them with undefined in there (yay JavaScript) so `item(index)` has a nullable return. We do the null assertions inside the test, which feels like the appropriate place.